bin/mixer_codegen.sh

```diff
@@ -31,7 +31,7 @@ set -e
 
 outdir=$ROOTDIR
 file=$ROOTDIR
-protoc="$ROOTDIR/bin/protoc.sh"
+protoc="protoc"
```

Comment:
fwiw, we switched to using the docker image to prevent requiring people to install their own version of `protoc` and to prevent having to manage versions, etc.

Reply:
Either way is fine. In CI we can pick binary since it makes code generation significantly faster.